src/reducers/snapshot.ts

```diff
@@ -279,10 +279,16 @@ export default {
           snapshotRequest: { $set: snapshotRequest },
         });
       },
-      [ActionTypes.PATCH_SNAPSHOT_DESCRIPTION_SUCCESS]: (state, action: any) =>
-        immutable(state, {
-          snapshot: { description: { $set: action.description } },
-        }),
+      [ActionTypes.PATCH_SNAPSHOT_SUCCESS]: (state, action: any) => {
+        const snapshotObj: any = { snapshot: {} };
+        if (action.data.consentCode !== undefined) {
```

Comment:
Would this cause a problem if they wanted to delete this field?

Reply:
RAS passport identifiers (a dataset’s PHS ID and a snapshot’s consent code) cannot be fully unset due to our choice of HTTP PATCH protocol for the underlying endpoint!